docs/xml/elements/ballot_measure_contest.rst

```diff
@@ -0,0 +1,83 @@
+|                  |                                                   |             |          |                                          |                        |
+|                  |                                                   |             |          |                                          |                        |
++------------------+---------------------------------------------------+-------------+----------+------------------------------------------+------------------------+
+| ProStatement     |:doc:`InternationalizedText                        | Optional    | Single   |Specifies a statement in favor of the     |If the element is       |
+|                  |<internationalized_text>`                          |             |          |referendum. It does not necessarily appear|invalid or not present, |
+|                  |                                                   |             |          |on the ballot.                            |the implementation is   |
+|                  |                                                   |             |          |                                          |required to ignore it.  |
+|                  |                                                   |             |          |                                          |                        |
++------------------+---------------------------------------------------+-------------+----------+------------------------------------------+------------------------+
+| SummaryText      |:doc:`InternationalizedText                        | Optional    | Single   |Specifies a short summary of the          |If the element is       |
+|                  |<internationalized_text>`                          |             |          |referendum that is on the ballot, below   |invalid or not present, |
+|                  |                                                   |             |          |the title, but above the text.            |the implementation is   |
+|                  |                                                   |             |          |                                          |required to ignore it.  |
+|                  |                                                   |             |          |                                          |                        |
+|                  |                                                   |             |          |                                          |                        |
++------------------+---------------------------------------------------+-------------+----------+------------------------------------------+------------------------+
+| Type             |:doc:`BallotMeasureType                            | Optional    | Single   |Specifies the particular type of ballot   |If the element is       |
```

Comment:
@pstenbjorn I might not have done it everywhere, but I tried link to the enumerations. If possible, I'd like to get this in and start making the fixes in subsequent pull requests.